src/frequenz/sdk/actor/power_distributing/_battery_status_tracker.py

```diff
@@ -179,7 +179,7 @@ def __init__(  # pylint: disable=too-many-arguments
         """Create class instance.
 
         Args:
-            battery_id: Id of this battery
+            component_id: Id of this battery
```

Comment:
The battery should remain in the documentation, right?

Reply:
yes, I guess it doesn't matter.  had to change the field to make it fit into a generic interface.